hack/install-vm.sh

```diff
@@ -0,0 +1,407 @@
+	    execute_command "$cmd" 2m
+	    ;;
+    esac
+}
+​
+# Install all pre-requisite packages needed for install
+install_docker_requisite_packages() {
+    set +eo pipefail
+    if ! command -v curl &> /dev/null; then
+        install_curl
+    fi
+    # Install docker, docker-compose only when running services in containers
+    if [ "$INSTALL_DOCKER" = true ]; then
+        docker version
+        if [[ "$?" -ne 0 ]]; then
+            echo "Docker is not installed"
+            install_container_runtime
+        fi
+        # Install docker-compose
+        docker-compose version
+        if [ "$?" -ne 0 ]; then
+            timeout --preserve-status --foreground 5s curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            if [ "$?" -eq 0 ]; then
+                echo "docker-compose is downloaded successfully"
+                chmod +x /usr/local/bin/docker-compose
+                ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+            else
+                echoerr "docker-compose couldn't be downloaded, Exiting!!"
+                exit 2
+            fi
+        fi
+    fi
+    set -eo pipefail
+}
+​
+# Create docker-compose file with ovs and antrea-agent container specific parameters
+create_docker_compose_file() {
+    cat << EOF > $AGENT_CONF_PATH/docker-compose.yaml
+version: "3"
+​
+services:
+  ovs:
+    image: $ANTREA_DOCKER_IMAGE
+    container_name: ovs
+    volumes:
+        - /var/log/openvswitch:/var/log/openvswitch
+        - /var/run/openvswitch:/var/run/openvswitch
+        - /lib/modules:/lib/modules
+    command: $OVS_BIN
+    network_mode: host
+    cap_add:
+        - NET_ADMIN
+        - SYS_NICE
+        - SYS_ADMIN
+        - IPC_LOCK
+​
+  antrea-agent:
+    image: $ANTREA_DOCKER_IMAGE
+    container_name: antrea-agent
+    volumes:
+        - /etc/antrea/:/etc/antrea
+        - /var/run/antrea:/var/run/antrea
+        - /var/run/openvswitch:/var/run/openvswitch
+        - /var/lib/cni:/var/lib/cni
+        - /var/log/antrea:/var/log/antrea
+        - /proc:/host/proc
+        - /var/run/netns:/host/var/run/netns
+        - /run/xtables.lock:/run/xtables.lock
+    environment:
+        - NODE_NAME=$NODE_NAME
+    command: $ANTREA_AGENT_BIN --config /etc/antrea/antrea-agent.conf --logtostderr=false --log_file=/var/log/antrea/antrea-agent.log
+    network_mode: host
+    privileged: true
+    cap_add:
+        - SYS_MODULE
+EOF
+}
+​
+load_ovs_modules() {
+    echo "Loading OVS kernel"
+    for module in "vport_geneve" "openvswitch"; do
+        lsmod | grep -q $module
+        if [[ $? -ne 0 ]]; then
+            echo "Load $module kernel module"
+            modprobe $module
+        else
+            echo "kernel module $module already loaded"
+        fi
+    done
+}
+​
+# Function to do the required job for starting antrea specfic containers
```

Comment:
Update the function description. 
start_containers - seems very generic, here we are only starting 2 containers using docker compose.
Try to give it a more meaningful name